blockchain/blockchain.go

```diff
@@ -317,12 +317,15 @@ func (bc *BlockChain) prefetchTxWorker(index int) {
 
 	logger.Debug("prefetchTxWorker is started", "index", index)
 	for followup := range bc.prefetchTxCh {
-		stateDB, err := state.New(bc.CurrentBlock().Root(), bc.stateCache)
+		stateDB, err := state.NewForPrefetching(bc.CurrentBlock().Root(), bc.stateCache)
 		if err != nil {
 			logger.Debug("failed to retrieve stateDB for prefetchTxWorker", "err", err)
 			continue
 		}
-		bc.prefetcher.PrefetchTx(followup.block, followup.ti, stateDB, bc.vmConfig, followup.followupInterrupt)
+		vmCfg := vm.Config{}
+		vmCfg = bc.vmConfig
```

Comment:
```suggestion
		vmCfg := bc.vmConfig
```
Is it intended? Or do I miss something? 😅

Reply:
Then, we need to declare the variable `vmCfg`, such as

```go
var vmCfg vm.Config
```